l3_dump.py

```diff
@@ -95,3 +117,7 @@ def exec_binary(cmdargs:list):
         elif DECODE_LOC_ID == 1:
             UNPACK_LOC = exec_binary([LOC_DECODER, '--brief', str(loc)])
             print(f"{tid=} {UNPACK_LOC} '{strings[offs]}' {arg1=} {arg2=}")
+
+        nentries += 1
+
+print(f"Unpacked {nentries=} log-entries.")
```

Comment:
FYI: I plan to grep on this message while building pytests (in future) to cross-check the working of this script with known log-file dumps.